Upstream changes

cibexgit opened the pull request:
@cibex/devs 👀

OCA-git-bot committed:
[UPD] README.rst

Commit:
[UPD] Update base_external_dbsource.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: server-backend-15.0/server-backend-15.0-base_external_dbsource
Translate-URL: https://translation.odoo-community.org/projects/server-backend-15-0/server-backend-15-0-base_external_dbsource/

dhx commented:
@ChristophAbenthungCibex seems the base_dav tests break here for some reason

ChristophAbenthungCibex commented:
@dhx This PR does not break the tests. The tests were broken already. I ran the tests locally without the PR and the same tests break. it seems the tests are broken for a while. I will have a look at them.

jdidderen committed:
base_user_role - add (copy) in order to allow the duplicate

OCA-git-bot committed:
Merge PR #237 into 15.0

Signed-off-by legalsylvain

Commit:
[UPD] Update base_user_role.pot

OCA-git-bot committed:
base_user_role 15.0.0.3.4

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Added translation using Weblate (Spanish)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: server-backend-15.0/server-backend-15.0-base_user_role
Translate-URL: https://translation.odoo-community.org/projects/server-backend-15-0/server-backend-15-0-base_user_role/
